docs/docs/reference-react-component.md

```diff
@@ -114,16 +114,16 @@ You can also return multiple items from `render()` using an array:
 ```javascript
 render() {
   return [
-    <li key="A"/>First item</li>,
-    <li key="B"/>Second item</li>,
-    <li key="C"/>Third item</li>,
+    <li key="A">First item</li>,
+    <li key="B">Second item</li>,
+    <li key="C">Third item</li>,
   ];
 }
 ```
 
 > Note:
 >
-> Don't forget to [add keys](/docs/lists-and-keys.html#keys) to elements in a fragment to avoid the key warning.
+> Don't forget to [add keys](https://facebook.github.io/react/docs/lists-and-keys.html#keys) to elements in a fragment to avoid the key warning.
```

Comment:
?

Reply:
Huh. No idea.

Reply:
That came from master.

https://raw.githubusercontent.com/facebook/react/master/docs/docs/reference-react-component.md

Reply:
This came in from `master` - looks like it was accidentally added via PR #10824

I've fixed it in this branch.